NEWS

```diff
@@ -6,7 +6,7 @@ UPDATE
 
 BUG
 * Fix ylab.axis.padding parameter in create.histogram
-* Fix error when using expressions in multipanelplot labels
+* Fix error when using expressions in multipanelplot labels and main
```

Comment:
What does "and main" mean?

Reply:
main is a synonym for 'title' and the name of the parameter in create.multipanelplot